src/openms/include/OpenMS/ANALYSIS/ID/IDBoostGraph.h

```diff
@@ -364,20 +364,58 @@ namespace OpenMS {
     /// Splits the initialized graph into connected components and clears it.
     void computeConnectedComponents();
 
+    /// @todo untested
+    /// Removes all edges from a peptide (and its PSMs) to its parent protein groups (and its proteins)
+    /// except for the best protein group.
+    /// @pre Graph must contain PeptideCluster nodes (e.g. with clusterIndistProteinsAndPeptides).
+    /// @param removeAssociationsInData Also removes the corresponding PeptideEvidences in the underlying
+    ///     ID data structure. Only deactivate if you know what you are doing.
+    void resolveGraphPeptideCentric(bool removeAssociationsInData = true);
+
 
 
     /// Zero means the graph was not split yet
     Size getNrConnectedComponents();
 
+    /// \brief Returns a specific connected component of the graph as a graph itself
```

Comment:
I don't mind the `\`' style, but its unusual in our codebase `@`
Maybe change?

Reply:
you are right, my IDE autocompletes to this style. I found a way to change it though! 👍